src/test/java/no/nav/pto/veilarbportefolje/persononinfo/PdlRepositoryTest.java

```diff
@@ -0,0 +1,137 @@
+    }
+
+    @Test
+    public void identOvergang_splitt() {
+        PDLIdent identIKonflikt = new PDLIdent("12345", true, AKTORID);
+        List<PDLIdent> identerBrukerA = List.of(
+                identIKonflikt,
+                new PDLIdent("12346", false, AKTORID),
+                new PDLIdent("00000", false, FOLKEREGISTERIDENT)
+        );
+        List<PDLIdent> identerBrukerB = List.of(
+                identIKonflikt,
+                new PDLIdent("12347", false, AKTORID),
+                new PDLIdent("00001", false, FOLKEREGISTERIDENT)
+        );
+        pdlRepository.upsertIdenter(identerBrukerA);
+        String lokalIdentBrukerA = pdlRepository.hentLokalIdent(identIKonflikt.getIdent());
+
+        var brukerAPreBrukerB = db.queryForList("select * from bruker_identer where person = ?", lokalIdentBrukerA)
+                .stream()
+                .map(PdlServiceTest::mapTilident)
+                .toList();
+
+        pdlRepository.upsertIdenter(identerBrukerB);
+
+        String lokalIdentBrukerB = pdlRepository.hentLokalIdent(identIKonflikt.getIdent());
+        var brukerAPostBrukerB = db.queryForList("select * from bruker_identer where person = ?", lokalIdentBrukerA)
+                .stream()
+                .map(PdlServiceTest::mapTilident)
+                .toList();
+        var brukerB = db.queryForList("select * from bruker_identer where person = ?", lokalIdentBrukerB)
+                .stream()
+                .map(PdlServiceTest::mapTilident)
+                .toList();
+
+        assertThat(lokalIdentBrukerA).isNotEqualTo(lokalIdentBrukerB);
+        assertThat(brukerAPreBrukerB).isEqualTo(identerBrukerA);
+        assertThat(brukerAPostBrukerB).isNotEqualTo(identerBrukerA);
+        assertThat(brukerB).isEqualTo(identerBrukerB);
+        assertThat(brukerAPostBrukerB.size()).isEqualTo(2);
+    }
+
+    @Test
+    public void oppfolgingAvsluttet_flereIdenterUnderOppfolging() {
+        AktorId historiskIdent = AktorId.of("12345");
+        AktorId ident = AktorId.of("12346");
+        List<PDLIdent> identer = List.of(
+                new PDLIdent(historiskIdent.get(), true, AKTORID),
+                new PDLIdent(ident.get(), false, AKTORID)
+        );
+        var historiskOpfolgingStart = new SisteOppfolgingsperiodeV1(null, historiskIdent.get(), ZonedDateTime.now(), null);
+        var nyOpfolgingStart = new SisteOppfolgingsperiodeV1(null, ident.get(), ZonedDateTime.now(), null);
+        var nyOpfolgingAvslutt = new SisteOppfolgingsperiodeV1(null, ident.get(), ZonedDateTime.now(), ZonedDateTime.now());
+        // Mock fix for oracle
+        oracle.update("insert into OPPFOLGINGSBRUKER (PERSON_ID, FODSELSNR) values (1234, '01010100000')");
+        oracle.update("insert into OPPFOLGINGSBRUKER (PERSON_ID, FODSELSNR) values (1235, '01010200000')");
+        oracle.update("insert into AKTOERID_TO_PERSONID (AKTOERID, PERSONID, GJELDENE) values ("+historiskIdent.get()+", 1235, 1)");
```

Comment:
Skal `GJELDENE` her være `1`? Har kanskje ikke noe å si for testen? I praksis skal vel `GJELDENDE` oppdateres når vi får endringer på Kafka-topic fra PDL? Mulig vi bør sjekke om `AKTOERID_TO_PERSONID` bør oppdateres også ved insert via start oppfølging?